src/lib/components/Input.tsx

```diff
@@ -130,7 +125,7 @@ const integerEnforcer = (nextUserInput: string) => {
   return null
 }
 export const IntegerInput = forwardRef(function IntegerInput(props: NumericInputProps, ref) {
-  return <NumericInput pattern="^[0-9]*$" enforcer={integerEnforcer} ref={ref as any} {...props} />
+  return <NumericInput pattern="^[0-9]*[.,]?[0-9]*$" enforcer={integerEnforcer} ref={ref as any} {...props} />
```

Comment:
`IntegerInput` should only allow integers. Revert this change, and use `DecimalInput` if you need decimals.